nessie/lakectl_test.go

```diff
@@ -104,9 +104,12 @@ func TestCommit(t *testing.T) {
 	RunCmdAndVerifyFailureWithFile(t, Lakectl()+" log lakefs://"+repoName+"/"+mainBranch, false, "lakectl_log_404", vars)
 	RunCmdAndVerifySuccessWithFile(t, Lakectl()+" repo create lakefs://"+repoName+" "+storage, false, "lakectl_repo_create", vars)
 	RunCmdAndVerifySuccessWithFile(t, Lakectl()+" log lakefs://"+repoName+"/"+mainBranch, false, "lakectl_log_initial", vars)
-	RunCmdAndVerifySuccessWithFile(t, Lakectl()+" fs upload -s files/ro_1k lakefs://"+repoName+"/"+mainBranch+"/ro_1k.1", false, "lakectl_fs_upload", vars)
+	RunCmdAndVerifySuccessWithFile(t, Lakectl()+" fs upload -s files/ro_1k.txt lakefs://"+repoName+"/"+mainBranch+"/ro_1k.txt.1", false, "lakectl_fs_upload", vars)
 	RunCmdAndVerifySuccessWithFile(t, Lakectl()+" log lakefs://"+repoName+"/"+mainBranch, false, "lakectl_log_initial", vars)
 	RunCmdAndVerifyFailureWithFile(t, Lakectl()+" commit lakefs://"+repoName+"/"+mainBranch, false, "lakectl_commit_no_msg", vars)
+	RunCmdAndVerifyFailureWithFile(t, Lakectl()+" commit lakefs://"+repoName+"/"+mainBranch+" -m \" \"", false, "lakectl_commit_empty_msg_err", vars)
+	RunCmdAndVerifySuccessWithFile(t, Lakectl()+" commit lakefs://"+repoName+"/"+mainBranch+" --allow-empty-message -m \" \"", false, "lakectl_commit_with_empty_msg_flag", vars)
+	RunCmdAndVerifySuccessWithFile(t, Lakectl()+" fs upload -s files/upload_file.txt lakefs://"+repoName+"/"+mainBranch+"/upload_file.txt.1", false, "lakectl_fs_second_upload", vars)
```

Comment:
I tyhink you can use the same source file and just (ro_1k) and just upload it to a different destination. That way you don't have to create a new file for uploading (the upload_file.txt)

Reply:
Absolutely!
Thanks